workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in undere a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data that you port to Middleware with advanced filtering options by date range and metric value.
+
+# How Dashboards Work
+
+The Dashboard Builder is a Graphical User Interface (GUI) based approach that enables users to create complex query logic without the added stress of learning a new query language. The metrics made available to users when constructing dashboards are dynamically generated based on the data that users port to Middleware. 
+
+# Create New Dashboard
+
+Create a new dashboard by selecting a data source to start your dashboard with:
```

Comment:
There's actually a step before this, to navigate to the Dashboard Builder page and hit "Create Dashboard". I would also use the wording, "To get started..." on any first step in the guides pages.

Reply:
We should also call out how there are two possible options here, either a) create dashboard from a data source, or b) creating a new widget from scratch, but to get started we recommend choosing data points from an existing source.